index.html

```diff
@@ -1554,12 +1554,13 @@
 definitions are likely to differ between domains.
 
 [=User agents=] tend to explain to [=people=] which [=origin=] or [=site=] provided the
-web page they're looking at. The [=party=] that controls this [=origin=] or [=site=] is
+web page they're looking at. The [=party=] that controls the use of information collected on an [=origin=] or [=site=] is
```

Comment:
I don't think that this is correct? The use of the information collected on an origin is often controlled by third parties. In fact, that's a key problem!